manifests/j/JetBrains/IntelliJIDEA/Community/2023.2.2/JetBrains.IntelliJIDEA.Community.installer.yaml

```diff
@@ -14,8 +14,11 @@ FileExtensions:
 - java
 - kt
 - kts
-ProductCode: IntelliJ IDEA Community Edition 2023.2.1
 ElevationRequirement: elevatesSelf
+AppsAndFeaturesEntries:
+- DisplayName: IntelliJ IDEA Community Edition
+  DisplayVersion: 2023.2.2
+  ProductCode: IntelliJ IDEA Community Edition

```

Comment:
```suggestion
AppsAndFeaturesEntries:
- DisplayName: IntelliJ IDEA Community Edition 2023.2.2
  DisplayVersion: 232.9921.47
  ProductCode: IntelliJ IDEA Community Edition 2023.2.2
```
These ***must*** match control panel exactly

Reply:
That's why I changed it:
![image](https://github.com/microsoft/winget-pkgs/assets/26790192/cd6a5e25-431b-4101-b1d7-d0df6c20fd05)

Reply:
> That's why I changed it: ![image](https://user-images.githubusercontent.com/26790192/272654470-cd6a5e25-431b-4101-b1d7-d0df6c20fd05.png)

Then there must be some discrepancy somewhere, because when I look at the `Add & Remove Programs` I see `232.9921.47`, which is what is written to the Registry
![image](https://github.com/microsoft/winget-pkgs/assets/12611259/5686b2a7-21b8-478d-b5cc-991858bbbff0)

Reply:
Interesting find. I myself can't find any record of the build number in the Registry. Only the `2023.2.2`. Seems like what everyone is used to from JetBrains 🙄. I'll have a deeper look.

Reply:
Ok. It seems this got mixed up on my system, because I installed IntelliJIDEA manually first and then switched to JetBrains "Toolbox" app manager, which instead of build numbers uses marketing version. Usually, it should also have its own separate product ID, but somehow - I suspect due to the switch - this was kept while the numberng scheme was changed. JB-Toolbox installs shouldn't show up in Winget and now after reinstalling without toolbox the `DisplayVersion` looks the same like yours on my end.